### ISSUE TITLE
Add vinted.nl

### ISSUE THREAD

**Issue by WebBreacher** [WebBreacher/WhatsMyName#412]
For this URL https://www.vinted.nl/member/general/search?search_text=john you will use the site's search since the links to the user profiles have random numbers in them.

**Comment by WebBreacher** [comment]
@C3n7ral051nt4g3ncy @K2SOsint either of you want to do this?

**Comment by C3n7ral051nt4g3ncy** [comment]
Will try to add

**Comment by K2SOsint** [comment]
@C3n7ral051nt4g3ncy send me a DM on Twitter if you need any assistance.
@WebBreacher is this the work-around for other forums as well when we encounter usernames with random numbers? Divert to the search options? If so, then I will add some more forums in the coming weeks...

**Comment by WebBreacher** [comment]
@K2SOsint yes. If possible, we try to find the exact user. If not, we can use the site's search and look for positive results. We use this with the archive.org site.

**Comment by C3n7ral051nt4g3ncy** [comment]
I am getting a false positive for vinted.

**Comment by WebBreacher** [comment]
Perhaps sharing your JSON would help us tweak it? And maybe sharing your script output?

**Comment by C3n7ral051nt4g3ncy** [comment]
{
           "name" : "vinted",
           "check_uri" : "https://www.vinted.nl/member/general/search?search_text={account}",
           "account_existence_code" : "200",
           "account_existence_string" : "userId",
           "account_missing_string" : "geen resultaten",
           "account_missing_code" : "200",
           "known_accounts" : ["john", "jack"],
           "category" : "business",
           "valid" : true
          }

**Comment by WebBreacher** [comment]
lets change the `account_existence_string` to `flag-box-flag-box-top` since `userId` appears in JavaScript.

**Comment by C3n7ral051nt4g3ncy** [comment]
Thanks for your help and input! Much appreciated

**Comment by WebBreacher** [comment]
Added with #435
